personalize.admin.inc

```diff
@@ -358,8 +367,12 @@ function personalize_agent_list($agents = NULL, $allow_edit = TRUE, $allow_alter
         $description = t('This table lists personalizations that use PHP (whenever possible) to display the fallback/winner variation for each variation set, which can cause variations to display more quickly than those in paused personalizations.');
         break;
     }
-    $build['status_' . $status] = array(
-      '#markup' => '<h2>' . $status_map[$status] . '</h2><p>' . $description . '</p>'
+    $build['personalization_group_' . $status] = array(
```

Comment:
Good to know. Done at [this line](https://github.com/acquia/personalize_module/commit/bad57b3e2ea9cbebdb782ab8938f318509265a5d#diff-336c0df322b81a28385f888327c4f932R375).